src/test/regress/expected/citus_locks.out

```diff
@@ -0,0 +1,22 @@
+-- store the current gpid in a variable
+SELECT global_pid AS coordinator_gpid
+FROM get_all_active_transactions()
+WHERE process_id = pg_backend_pid() \gset
+-- list the locks on relations for current distributed transaction
+SELECT relation::regclass, nodeid, mode, granted
+FROM citus_locks
+WHERE global_pid = :coordinator_gpid AND locktype = 'relation'
+ORDER BY 1, 2, 3, 4;
+              relation              | nodeid |      mode       | granted
+---------------------------------------------------------------------
+ pg_locks                           |      2 | AccessShareLock | t
+ pg_dist_partition                  |      2 | AccessShareLock | t
+ pg_dist_partition                  |      2 | AccessShareLock | t
+ pg_dist_node                       |      2 | AccessShareLock | t
+ pg_dist_node_pkey                  |      2 | AccessShareLock | t
+ pg_dist_node_nodename_nodeport_key |      2 | AccessShareLock | t
+ pg_dist_local_group                |      2 | AccessShareLock | t
+ pg_dist_local_group_pkey           |      2 | AccessShareLock | t
+ citus_locks                        |      2 | AccessShareLock | t
+(9 rows)
```

Comment:
I'm not sure what this test does. Can you do something like:
```SQL
BEGIN;
ALTER TABLE dist_table ...;
SELECT * FROM citus_locks WHERE dist_table;
```

Reply:
I did a similar thing after an `ALTER TABLE` command on a distributed table in a transaction block.